packages/docs/src/pages/components/switch/examples/ExSizes.vue

```diff
@@ -14,3 +14,15 @@
         </b-field>
     </section>
 </template>
+
+<script lang="ts">
```

Comment:
You can simplify these lines as:
```vue
<script setup lang="ts">
import { BField, BSwitch } from '@ntohq/buefy-next'
</script>
```

Reply:
@wesdevpro Unfortunately, type-checking won't work with your code. For instance, we expect changing the line 4 in this file to the following would produce a type error because the `size` prop requires a string:
```vue
<b-switch :size="123">Small</b-switch>
```
However, no type error will be detected with your code.

The keyword `setup` in my snippet is important here, which enables the [composition API style](https://vuejs.org/guide/introduction.html#composition-api).

Reply:
Can you explain why no type error will be caught by the type script compiler?

Reply:
For Vue to perform type-checking, all the components have to be locally registered. Your code in the commit fa9353355eec8cb820c1173c31884f1d2a500b89 exported an empty component, `export default {}`, which has neither `BField` nor `BSwitch` locally registered. Simply importing `BField` and `BSwitch` in the [options API style](https://vuejs.org/guide/introduction.html#options-api) does not register them.

On the other hand, in the [composition API style](https://vuejs.org/guide/introduction.html#composition-api) (with `setup`), imported components are automatically registered.

The `setup` keyword is not a requirement, in fact, your initial commit 66654946a84acae09c027bf749e568c7dfc678a8 worked.